### DISCUSSION TITLE
Info on raw dataset preprocessing

### DISCUSSION THREAD

**Discussion by agoliaei** [mims-harvard/TDC#195] [Q&A]
Hi,
Thanks so much for sharing this package with the community! 
I was wondering, where can I find more information on how raw datasets are preprocessed? For example, bindingdb_ic50, bindingdb_kd, and bindingdb_ki: I want to know the criteria that was used to select examples from the original raw bindingdb dataset. For example: are all selected examples human proteins? what was the date of the original download? are proteins sequences filtered by length? ....
If someone could point me to the file where this preprocessing is done, that would be great.
Thanks again for sharing this amazing package!

**Comment by kexinhuang12345** [comment]
Sorry for the late reply! It seems there is no notification for me. Here is the repo where we post the raw processing scripts: https://github.com/kexinhuang12345/data_process
